lib/workers/branch/commit.js

```diff
@@ -30,15 +30,13 @@ async function commitFilesToBranch(config) {
       logger.info('DRY-RUN: Would commit files to branch ' + config.branchName);
     } else {
       // API will know whether to create new branch or not
-      const res = await platform.commitFilesToBranch(
+      await platform.commitFilesToBranch(
         config.branchName,
         updatedFiles,
         config.commitMessage,
         config.baseBranch || undefined
       );
-      if (res) {
-        logger.info({ branch: config.branchName }, `Branch ${res}`);
```

Comment:
but the `Storage.commitFilesToBranch` has no `return, so it's `void`! It will throw it there are any errors. so we can always log success if we are here.

https://github.com/renovatebot/renovate/blob/e57bfef77765c7e72fa91861db364fd0669c6765/lib/platform/git/storage.ts#L378-L432

Reply:
https://github.com/renovatebot/renovate/blob/master/lib/platform/github/index.ts#L536-L548